### PR TITLE
Rename ads, alm, dlm to EdgeFarm.data, EdgeFarm.applications and EdgeFarm.devices

### PR DESCRIPTION
**Pull request by KairaMiu**

### CODE REVIEW

File: cli/internal/configuration/configuration.go [hunk 1]
```diff
@@ -20,9 +20,9 @@ import "fmt"
 
 const (
 	// DefaultDlmServer Production URL to edgefarm-dlm-server
-	DefaultDlmServer = "https://dlm.edgefarm.io/v1"
+	DefaultDlmServer = "https://devices.edgefarm.io/v1"
 	// DefaultAlmServer Production URL to edgefarm-alm-server
-	DefaultAlmServer = "https://alm.edgefarm.io/v1"
+	DefaultAlmServer = "https://applications.edgefarm.io/v1"
```
Comment (siredmar) [inline cli/internal/configuration/configuration.go:25]: Please rename DefaultDlmServer to DefaultDevicesServer 
Please rename DefaultAlmServer to DefaultApplicationsServer

Comment (KairaMiu) [reply cli/internal/configuration/configuration.go:25]: ok

File: cli/cmd/root.go [hunk 2]
```diff
@@ -69,8 +69,8 @@ func initConfig() {
 	// priority 1: settings from config file
 	// priority 2: default settings
 	// If a config file is found, read it in.
-	viper.SetDefault("dlm_server_url", configuration.DefaultDlmServer)
-	viper.SetDefault("alm_server_url", configuration.DefaultAlmServer)
+	viper.SetDefault("dlm_server_url", configuration.DefaultDevicesServer)
+	viper.SetDefault("alm_server_url", configuration.DefaultApplicationsServer)
```
Comment (siredmar) [inline cli/cmd/root.go:73]: dlm_server_url and alm_server_url

Comment (KairaMiu) [reply cli/cmd/root.go:73]: ok

**Review by siredmar — APPROVED** [review]
Looks fine to me for now.

**Pull request merged**